# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1322627640 28800
#      Tue Nov 29 20:34:00 2011 -0800
# Node ID efa658122cc92b7124d19a0d1d8cac101b176291
# Parent  0fea4cf22f88bc7debc6dc6cc62c73df744b78d3
Fix tilde_expand %!test bug on MinGW (Bug #33862)

* sysdepd.cc (tilde_expand): Change test to match function behavior.

diff --git a/src/sysdep.cc b/src/sysdep.cc
--- a/src/sysdep.cc
+++ b/src/sysdep.cc
@@ -891,12 +891,12 @@ tilde_expand (\"~/bin\")\n\
 }
 
 /*
 %!test
 %! if (isempty (getenv ("HOME")))
 %!   setenv ("HOME", "foobar");
 %! endif
 %! home = getenv ("HOME");
-%! assert (tilde_expand ("~/foobar"), [home filesep() "foobar"]);
+%! assert (tilde_expand ("~/foobar"), strcat (home, "/foobar"));
 %! assert (tilde_expand ("/foo/bar"), "/foo/bar");
 %! assert (tilde_expand ("foo/bar"), "foo/bar");
 */
